test/model/src/driver/kv_workload.cpp

```diff
@@ -37,6 +37,168 @@ extern "C" {
 
 namespace model {
 
+namespace operation {
+
+/*
+ * parse --
+ *     Parse an operation from a string. Throw an exception on error.
+ */
+any
+parse(const char *str)
+{
+    const char *p = str;
+
+    /* Get the operation name. */
+    std::string name;
+    while (*p != '\0' && (std::isalnum(*p) || *p == '_'))
```

Comment:
This is very inefficient. A new std::string is created for each character. It would be better to search and then call std::string::substr() once. The search itself can be optimized by using std::string::find_first_of() and std::string:find_first_not_of().

Reply:
I improved this using `std::string_view` which seems to be even most efficient! I did not end up using `find_first_not_of`, as it doesn't have a concise way to specify all letters and numbers. Fixed in [70241f1](https://github.com/wiredtiger/wiredtiger/pull/10267/commits/70241f1a1fcf8c5b60f1e9303b46e4568d643a27).